core/bignum/divmod_spec.rb

```diff
@@ -14,10 +14,13 @@
     @bignum.divmod(4).should == [2305843009213693965, 3]
     @bignum.divmod(13).should == [709490156681136604, 11]
 
-    @bignum.divmod(4.0).should == [2305843009213693952, 0.0]
-    @bignum.divmod(13.0).should == [709490156681136640, 8.0]
+    not_supported_on :opal do
+      @bignum.divmod(4.0).should == [2305843009213693952, 0.0]
+      @bignum.divmod(13.0).should == [709490156681136640, 8.0]
+
+      @bignum.divmod(2.0).should == [4611686018427387904, 0.0]
```

Comment:
See lines above. Ruby does a diverent divmod when a Bignum is divided by a Float.
But 4.0 is a Numeric an a Fixnum in Opal so it is simply 4. 
Same for 13.0 and 2.0.